### DISCUSSION TITLE
How can I use ddev config without waiting for the prompts?

### DISCUSSION THREAD

**Discussion by SimonWpt** [ddev/ddev#2685] [Q&A] [answered]
Is there an elegant way to run `ddev config` after an already done configuration without pressing `ENTER` 3 times?

**Comment by rfay** [comment]
`ddev config --php-version=7.3` or `ddev config --project-type=drupal9` or with any flag. 

It's been suggested to add a --auto flag or something like that that would just skip the prompts. Never seems to have reached the level of a formal request though.

**Comment by KarimBoudjema** [comment, marked as answer]
You can use flags, in the following example we create the config for a Drupal8 site with a web docroot.
`ddev config --project-type=drupal8 --docroot=web --create-docroot --project-name=my-project`
